lv_conf_template.h

```diff
@@ -260,79 +272,87 @@
  * Logging
  *-----------*/
 
-/*Enable the log module*/
+/** Enable log module */
 #define LV_USE_LOG 0
 #if LV_USE_LOG
-
-    /*How important log should be added:
-    *LV_LOG_LEVEL_TRACE       A lot of logs to give detailed information
-    *LV_LOG_LEVEL_INFO        Log important events
-    *LV_LOG_LEVEL_WARN        Log if something unwanted happened but didn't cause a problem
-    *LV_LOG_LEVEL_ERROR       Only critical issue, when the system may fail
-    *LV_LOG_LEVEL_USER        Only logs added by the user
-    *LV_LOG_LEVEL_NONE        Do not log anything*/
+    /** Set value to one of the following levels of logging detail:
+     *  - LV_LOG_LEVEL_TRACE    Log detailed information.
+     *  - LV_LOG_LEVEL_INFO     Log important events.
+     *  - LV_LOG_LEVEL_WARN     Log if something unwanted happened but didn't cause a problem.
+     *  - LV_LOG_LEVEL_ERROR    Log only critical issues, when system may fail.
+     *  - LV_LOG_LEVEL_USER     Log only custom log messages added by the user.
+     *  - LV_LOG_LEVEL_NONE     Do not log anything. */
     #define LV_LOG_LEVEL LV_LOG_LEVEL_WARN
 
-    /*1: Print the log with 'printf';
-    *0: User need to register a callback with `lv_log_register_print_cb()`*/
+    /** - 1: Print log with 'printf';
+     *  - 0: User needs to register a callback with `lv_log_register_print_cb()`. */
     #define LV_LOG_PRINTF 0
 
-    /*Set callback to print the logs.
-     *E.g `my_print`. The prototype should be `void my_print(lv_log_level_t level, const char * buf)`
-     *Can be overwritten by `lv_log_register_print_cb`*/
+    /** Set callback to print logs.
+     *  E.g `my_print`. The prototype should be `void my_print(lv_log_level_t level, const char * buf)`.
+     *  Can be overwritten by `lv_log_register_print_cb`. */
     //#define LV_LOG_PRINT_CB
 
-    /*1: Enable print timestamp;
-     *0: Disable print timestamp*/
+    /** - 1: Enable printing timestamp;
+     *  - 0: Disable printing timestamp. */
     #define LV_LOG_USE_TIMESTAMP 1
 
-    /*1: Print file and line number of the log;
-     *0: Do not print file and line number of the log*/
+    /** - 1: Print file and line number of the log;
+     *  - 0: Do not print file and line number of the log. */
     #define LV_LOG_USE_FILE_LINE 1
 
+    /* Enable/disable LV_LOG_TRACE in modules that produces a huge number of logs. */
 
-    /*Enable/disable LV_LOG_TRACE in modules that produces a huge number of logs*/
+    /** Enable/disable trace logs in memory operations. */
     #define LV_LOG_TRACE_MEM        1
+    /** Enable/disable trace logs in timer operations. */
```

Comment:
Can we make it inline with the define to save some space?

Reply:
@kisvegabor Great idea!  Done and submitted.